WikiWatershed/model-my-watershed
# Monitor: Remove Search Data Button in BiG-CZ

## Overview

Removes "Search Data" nav button in BiG-CZ which led to a now-broken state. Since all the search functionality has been subsumed in the Monitor tab beside Analyze, we no longer need the extra stage.

Also removes unused controllers, views, and templates.

Connects #2639 

### Demo

![2018-03-05 15 26 52](https://user-images.githubusercontent.com/1430060/36998265-4bea1bf6-208a-11e8-951a-22781ca81542.gif)

Tagging @jfrankl and @ajrobbins for visual review.

### Notes

Should the "Monitor" tab be "Search" or "Data" or something in BiG-CZ? It was previously called "Data" in this UI:

![image](https://user-images.githubusercontent.com/1430060/36998542-42576f48-208b-11e8-8ae3-9187e1043992.png)

## Testing Instructions

* Checkout this branch and `bundle`
* Go to [:8000/bigcz](http://localhost:8000/?bigcz) and proceed to Analyze
* Ensure you don't see a "Search Data" button in the sidebar footer, and the "Change Area" button is now full width
* Ensure you can search data from the Monitor tab
* Ensure you can Change Area, select a different shape, and start a new search in the Monitor tab
* Go to [:8000/](http://localhost:8000/) and ensure Analyze and Monitor look and work as expected
